README.md

```diff
@@ -1,85 +1,88 @@
-MicroPython Modules and Samples for XBee Devices
-================================================
+MicroPython Resources for XBee Devices
+======================================
 
-This project contains modules and sample code for use on XBee devices
-with MicroPython.  [Digi International][Digi] manages the project on
-GitHub as [xbee-micropython].
+This project contains modules and sample code for use on XBee devices with
+MicroPython. Digi has written some of the modules and samples, in addition to
+modifying existing code to address differences between the XBee and other
+MicroPython platforms. This project includes code from [micropython-lib].
 
-Digi has written some of the modules and samples, in addition to modifying
-existing code to address differences between the XBee and other MicroPython
-platforms.  This project includes code from [micropython-lib].
+The repository is used by the **XBee MicroPython plug-in for PyCharm** to
+get the available samples, libraries platforms and stubs and facilitate the
+process of creating and launching MicroPython applications in XBee devices.
+This means that you don't need to clone it unless you wan't to contribute
+with new content as the PyCharm plug-in will handle all necessary resources
+automatically.
 
-Please use GitHub to open issues and submit pull requests with improvements
-or useful code to share with other users.
+We don't recommend you to do so, but you can still use the content of this
+repository to create XBee MicroPython applications by your own (without using
+the **XBee MicroPython plug-in for PyCharm**). In any case, you will find
+information on how to get started with XBee and MicroPython in the
+[Digi MicroPython Programming Guide][doc].
 
-[Digi]: http://www.digi.com
-[xbee-micropython]: https://github.com/digidotcom/xbee-micropython
-[micropython-lib]: https://github.com/micropython/micropython-lib
+[Digi International][Digi] manages the project on GitHub as [xbee-micropython].
 
 
 Requirements
 ------------
 
-The following XBee devices include support for MicroPython, but not
-a file system or modules:
-
-  * XBee and XBee3 Cellular (firmware *07 and later)
-  * XBee3 Zigbee (firmware 1003 and later)
-
-The following XBee devices include file system and module support:
+Modules and samples within the repository can be executed only in XBee
+devices with MicroPython support. This is the list of current compatible
+devices:
 
-  * XBee and XBee3 Cellular (firmware *0B and later)
+* Digi XBee3 Zigbee 3 (firmware 1006 and later)
+* Digi XBee3 802.15.4 (firmware 2003 and later)
+* Digi XBee3 DigiMesh 2.4 (firmware 3002 and later)
+* Digi XBee3 Cellular LTE-M/NB-IoT (firmware 11410 and later)
+* Digi XBee3 Cellular LTE CAT 1 (firmware 31010 and later)
```

Comment:
What about "Digi XBee Cellular LTE CAT 1 Verizon (firmware 100B and later)", and "Digi XBee Cellular 3G Global (firmware 1130B and later)"?

Reply:
Is the MicroPython API of these modules in line with the other XBee3 Cellular variants? I mean, do they have the same Python modules and features? It's just to know if we should add them as compatible platforms in all the projects or not.

Reply:
The differences in MicroPython between "XBee Cellular" (EFM32) and "XBee3 Cellular" (EFR32) are:

- XBee3 Cellular supports `digi.cloud`.
- XBee3 Cellular `xbee.relay` module supports the Bluetooth interface.

Otherwise they are all identical.

Reply:
Thanks!, I've updated the samples readmes.

Reply:
Doesn't look like you updated this README though.

Reply:
Directory READMEs were missing them, it's fixed now, thanks!